website/content/en/preview/concepts/nodepools.md

```diff
@@ -167,6 +136,18 @@ spec:
   # to select. Higher weights indicate higher priority when comparing NodePools.
   # Specifying no weight is equivalent to specifying a weight of 0.
   weight: 10
+status:
```

Comment:
Should we add a status section that provides an explanation of each filed? Similar to how it's done for nodeclasses?

Reply:
I went through and made sure that all fields are now described.